CHANGELOG.md

```diff
@@ -96,6 +97,7 @@
 * [BUGFIX] Native histograms: fix how IsFloatHistogram determines if mimirpb.Histogram is a float histogram. #4706
 * [BUGFIX] Query-frontend: fix query sharding for native histograms. #4666
 * [BUGFIX] Ring status page: fixed the owned tokens percentage value displayed. #4730
+* [BUGFIX] Querier: fixed chunkIterator may return values after Seek beyond end of chunk. #4450
```

Comment:
This description is too low-level. How did bug manifest for the user?

Reply:
If I understand that correctly, the bug can cause the query to return the next chunk that might be not part of the query. However @krajorama should have better context on this. Issue: https://github.com/grafana/mimir/issues/4415.

If this is not considered changelog-worthy, I can also remove this.

Reply:
So according to @krajorama next chunk will be the sample in the next timestamp.

Reply:
It did not manifest, we realized this theoretical situation during code review.

Reply:
Since it did not happen, I'm ok with omitting from changelog